src/Core/Tools/SendFeatures/Services/SendFileSettingHelper.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Bit.Core.Tools.SendFeatures;
+﻿using Bit.Core.Tools.Entities;
```

Comment:
❓ Why wasn't this import needed before?

Reply:
JetBrains Rider IDE didn't pick it up beforehand. I'm not sure why it didn't. It is needed for the XMLdocs comment if you want to be able to click the 'Send' type and it take you to the correct type.